src/config.js

```diff
@@ -8,6 +8,11 @@ export class Config {
   configurations = {}
 
   constructor(viewManager, viewManagerConfig) {
+    this.configure({
+      labelFormat: element => {
+        return element.label || element.key;
+      }
```

Comment:
you do like to use return, do you :smile: ?
i'd use just 
`labelFormat: element => element.label || element.key`
but imho in the end, it's just a matter of taste